### PR TITLE
Add Polish translation

### PR DESCRIPTION
**Pull request by pkomur**

### CODE REVIEW

File: shell-volume-mixer@derhofbauer.at/locale/pl/LC_MESSAGES/gnome-shell-extensions-shell-volume-mixer.po [hunk 1]
```diff
@@ -0,0 +1,112 @@
+# German translation for GNOME Shell Volume Mixer.
```
Comment (aleho) [inline shell-volume-mixer@derhofbauer.at/locale/pl/LC_MESSAGES/gnome-shell-extensions-shell-volume-mixer.po]: Make this "Polish" :D

File: shell-volume-mixer@derhofbauer.at/locale/pl/LC_MESSAGES/gnome-shell-extensions-shell-volume-mixer.po [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+# Copyright (C) 2014
+# This file is distributed under the same license as the GNOME Shell Volume Mixer.
+# Jonatan Zedler <jonatan_zedler@gmx.de>, 2014.
+# Alexander Hofbauer <alex@derhofbauer.at>, 2014.
```
Comment (aleho) [inline shell-volume-mixer@derhofbauer.at/locale/pl/LC_MESSAGES/gnome-shell-extensions-shell-volume-mixer.po]: Give yourself some credit. :D

**Comment by pkomur** [comment]
Thanks. Done.

**Comment by aleho** [comment]
Would you mind rebasing it onto latest master?

There's a new feature and new strings as well.

**Comment by pkomur** [comment]
Please generate template POT file.

**Comment by pkomur** [comment]
Commit new pull request #73

**Pull request closed without merging**